### PR TITLE
Add crud functionality to the To-do-List Project

### PR DESCRIPTION
**Pull request by tariktesfa**
In this PR, a crud operation is added to the To-do-list project. The to-do list project can 

- Add a new Task to the to-do list
- Update the already existing lists and
- Remove a task from the list

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+const onSubmit = () => {
+  const addTodoInput = document.querySelector('#add-todo');
+  if (!addTodoInput.value) {
+    return;
+  }
+  const newItem = {
+    description: addTodoInput.value,
+    completed: false,
+    index: todoList.data.length,
```
Comment (leolpaz) [inline src/crud.js:48]: - [ ] As pointed by the requirements, the index of the task should be the **new** array length which is the array length + 1, kindly fix that so it matches the example given in the requirement:
image

Yours:

image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @tariktesfa , 

You've done an amazing job so far, I'd like to highlight the following:
- Great job with the add and remove task
- Linter is passing

Some issues need to be resolved before your approval, but you're almost there!




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js (hunk 1 above)
Comment (tariktesfa) [reply src/crud.js:48]: I really appreciate your comment and now I resolved it.

**Review by descholar-ceo — APPROVED** [review]
Hi @tariktesfa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**